nilearn/maskers/nifti_labels_masker.py

```diff
@@ -504,6 +504,41 @@ def fit(self, imgs=None, y=None):
         msg = f"loading data from {repr}"
         _utils.logger.log(msg=msg, verbose=self.verbose)
         self.labels_img_ = _utils.check_niimg_3d(self.labels_img)
+
+        # create _region_id_name dictionary
+        # this dictionary will be used to store region names and
+        # the corresponding region ids as keys
+        self._region_id_name = None
```

Comment:
```suggestion
        self._region_id_name = {}
```
this way we know this is always a dict